### PR TITLE
NEW Session grant files without permissions

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-sharedraftcontent/issues/132

Needs https://github.com/silverstripe/silverstripe-assets/pull/465 merged before travis will work

### CODE REVIEW

**Review by emteknetnz — COMMENTED** [review]
Should target 2 since there's new API

**Review by andrewandante — APPROVED** [review]
Looks good to me 😄

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,6 @@ import:
 
 env:
   global:
+    # Change to "4.10.x-dev || " once the 4.10 branch in silverstripe/assets has been created
+    - REQUIRE_RECIPE="4.x-dev || "
```
Comment (michalkleiner) [inline .travis.yml]:
```suggestion
    - REQUIRE_RECIPE="4.10.x-dev || "
```

Comment (michalkleiner) [reply .travis.yml]: Framework 4.10 and Assets 1.10 do exist now, should we adjust here?

**Review by michalkleiner — CHANGES_REQUESTED** [review]

**Review by michalkleiner — APPROVED** [review]
Merge on green

**Pull request merged**